picoCTF-web/api/user.py

```diff
@@ -351,8 +353,14 @@ def disable_account(uid):
     db = api.db.get_conn()
     db.users.find_one_and_update({
         "uid": uid,
+        "disabled": False
     }, {"$set": {
-        "disabled": True
+        "disabled": True,
+        "firstname": "null",
```

Comment:
Let's set these to empty strings rather than the string "null". That's the convention for missing fields in the database currently.